spec/views/sign_up/completions/show.html.erb_spec.rb

```diff
@@ -90,4 +90,13 @@
       expect(rendered).to include('9**-**-***6')
     end
   end
+
+  context 'mfa banner' do
+    let(:user) { create(:user, :signed_up, :with_webauthn_platform, :with_phone) }
+    it 'does not show a banner if multiple factors are enabled' do
+
+      render
+      expect(rendered).not_to have_content(I18n.t('multi_factor_authentication.second_method_warning.text'))
+    end
+  end
```

Comment:
Since the view doesn't deal with the user, but rather an instance variable set by the controller, I might expect the stubbing to occur with the `@multiple_factors_enabled` variable, not the user (where user stubbing could happen in the controller tests). Also, we're only testing one of the two possible variations of this behavior.

Maybe something like...

```suggestion
  context 'mfa banner' do
    let(:multiple_factors_enabled) { false }

    before do
      @multiple_factors_enabled = multiple_factors_enabled
    end

    it 'shows a banner' do
      render

      expect(rendered).to have_content(t('multi_factor_authentication.second_method_warning.text'))
    end

    context 'with multiple factors enabled' do
      let(:multiple_factors_enabled) { true }

      it 'does not show a banner' do
        render

        expect(rendered).not_to have_content(t('multi_factor_authentication.second_method_warning.text'))
      end
    end
  end
```